site/src/pages/tutorials/grpc/blog/define-service.mdx

```diff
@@ -0,0 +1,108 @@
+---
+menuTitle: "Define service"
+order: 1
+type: step
+targetLang: java
+---
+
+# Defining a blog service and messages
+
+Let's begin by defining our gRPC blog service in a proto file.
+
+<TutorialSteps current={1} />
+
+## 1. Create a proto file
+
+Create a file, `blog.proto` inside the directory, `{project_root}/src/main/proto`. This tutorial uses [Protocol Buffers version 3](https://developers.google.com/protocol-buffers/docs/proto3).
+
+```protobuf filename=blog.proto
+syntax = "proto3";
+
+package example.armeria.server.blog.grpc;
+option java_package = "example.armeria.server.blog.grpc";
```

Comment:
Let's remove `server` because we use this on the client-side as well.
You can just run this command:
`find . -name '*.mdx' -print0 | xargs -0 sed -i '' -e 's/example\.armeria\.server\.blog\.grpc/example\.armeria\.blog\.grpc/g'`

Reply:
Both sound good.